fem/gslib.cpp

```diff
@@ -117,27 +117,40 @@ FindPointsGSLIB::FindPointsGSLIB()
    crystal_init(cr, gsl_comm);
 }
 
+FindPointsGSLIB::FindPointsGSLIB(Mesh *mesh_in, const double bb_t,
+                                 const double newt_tol, const int npt_max)
+   : FindPointsGSLIB()
+{
+   Setup(*mesh_in, bb_t, newt_tol, npt_max);
```

Comment:
Here `mesh_in` can't be `nullptr`, so better take as reference in the declaration to make it clear.